### PR TITLE
Term entry sql log10

### LINKED ISSUE #2260: [Term Entry] SQL Math Functions LOG10
### Reason/inspiration (optional)

We would like a new term entry for the Math Function, `LOG10`, in SQL.

The entry should go in a file in docs/content/sql/concepts/math-functions/terms/log10/log10.md

The entry should: 

- Explain what the math function `LOG10` is in SQL and what it’s used for, starting with an introductory paragraph (2-3 sentences).
- Include SQL database compatibility as a note.
- Links out to entries as needed.

Please refer to the [entry templates (term entry template in particular)](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added a new term for SQL: Math Functions - log10
- Closes #2260 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/log10/log10.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+| 3       | 100          |
+
+The `LOG10()` function can calculate the base-10 logarithm of each number in the `POWER_OF_TEN` table:
+
+```sql
+SELECT num_id, LOG10(num_10) AS num_log10
+FROM POWER_OF_TEN;
+```
+
+This example results in the following output:
```
Comment (CBID2) [inline content/sql/concepts/math-functions/terms/log10/log10.md:52]: [Proofreading]
```suggestion
The above example results in the following output:
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @cslylla! :) I just gave a proofreading recommendation for your entry, so check that out to learn how to improve it. Other than that, you're off to a great start! :)

File: content/sql/concepts/math-functions/terms/log10/log10.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Syntax
+
+```sql
+LOG10(number)
+```
+
+The `LOG10()` function takes one required parameter, `number`:
+
+- a `float` type number greater than `0`, or
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/log10/log10.md:28]:
```suggestion
- A `float` type number greater than `0`.
```

File: content/sql/concepts/math-functions/terms/log10/log10.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- an expression resulting in a `float` type number (e.g. `EXP(10)` or `10*10`).
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/log10/log10.md:29]:
```suggestion
- An expression resulting in a `float` type number (e.g. `EXP(10)` or `10*10`).
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've added a couple of comments. Please update the entry, and tag me when you've made the changes. Thanks

**Comment by cslylla** [comment]
> Hi @cslylla! :) I just gave a proofreading recommendation for your entry, so check that out to learn how to improve it. Other than that, you're off to a great start! :)

Hi @CBID2, thanks for the feedback! I have just approved the suggested change! 😊

**Comment by cslylla** [comment]
@caupolicandiaz I approved the changes; thank you!

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good to merge.

**Comment by SSwiniarski** [comment]
@cslylla Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/sql/math-functions/log10

image

**Pull request merged**